cmake/compile_definitions/macos.cmake

```diff
@@ -8,6 +8,7 @@ link_directories(/opt/homebrew/lib)
 ADD_DEFINITIONS(-DBOOST_LOG_DYN_LINK)
 
 list(APPEND SUNSHINE_EXTERNAL_LIBRARIES
+        ${APP_KIT_LIBRARY}
```

Comment:
Could you also re-order the `FOUNDATION_LIBRARY`? I know it was out of order before, but since you are modifying the code here it could be done at the same time.